mono/metadata/loaded-images-internals.h

```diff
@@ -0,0 +1,67 @@
+/**
+* \file
+*/
+
+#ifndef _MONO_METADATA_IMAGE_HASHES_H_
+#define _MONO_METADATA_IMAGE_HASHES_H_
+
+#include <glib.h>
+#include <mono/metadata/object-forward.h>
+#include <mono/metadata/loader-internals.h>
+#include <mono/utils/mono-forward.h>
+#include <mono/utils/mono-error.h>
+
+/*
+ * The "loaded images" hashes keep track of the various assemblies and netmodules loaded
+ * There are four, for all combinations of [look up by path or assembly name?]
+ * and [normal or reflection-only load?, as in Assembly.ReflectionOnlyLoad]
+ */
+enum {
+	MONO_LOADED_IMAGES_HASH_PATH = 0,
+	MONO_LOADED_IMAGES_HASH_PATH_REFONLY = 1,
+	MONO_LOADED_IMAGES_HASH_NAME = 2,
+	MONO_LOADED_IMAGES_HASH_NAME_REFONLY = 3,
+	MONO_LOADED_IMAGES_HASH_COUNT = 4
+};
+
+struct _MonoLoadedImages {
+	MonoAssemblyLoadContext *owner; /* NULL if global */
```

Comment:
To make sure I'm clear on this, it's only global on legacy, right?

Reply:
Yes.